spmd/tensor/__init__.py

```diff
@@ -4,7 +4,7 @@
 import torch
 from spmd.tensor.api import DTensor
 from spmd.tensor.device_mesh import DeviceMesh, get_global_device_mesh
-from spmd.tensor.placement_types import Placement, Shard, Replicate, _Partial
+from spmd.tensor.placement_types import Placement, Shard, Replicate
```

Comment:
Is it possible we only allow user to import these three from just one place? Or this is just internal and in public users should always do from spmd.tensor import Placement, Shard, Replicate?

Another question, will this be part of TP beta release? I guess this belongs to DTensor right?

Reply:
It's internal and in public user always suppose to import from `spmd.tensor` instead of submodules.

For the second question, no, I think TP beta release should not expose the placement types.